### PR TITLE
[Improve][Connector-v2-Fake]Supports direct definition of data values(row)

### PR DESCRIPTION
**Pull request by laglangyue**
## Purpose of this pull request

subtask of #2752 

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: docs/en/connector-v2/source/FakeSource.md [hunk 1]
```diff
@@ -22,15 +22,18 @@ just for testing, such as type conversion and feature testing
...
 ### result_table_name [string]
 
 The table name.
 
 ### type [string]
+
 Table structure description ,you should assign schema option to tell connector how to parse data to the row you want.  
 **Tips**: Most of Unstructured-Datasource contain this param, such as LocalFile,HdfsFile.  
 **Example**:
+
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/FakeSource.md:36]: Please add some comment of the new params `row.num`

**Comment by EricJoy2048** [comment]
Please add the e2e test for the new params `row.num`.

**Comment by laglangyue** [comment]
image

**Comment by laglangyue** [comment]
> Please add the e2e test for the new params `row.num`.

done

**Comment by laglangyue** [comment]
@hailin0 @EricJoy2048 PTAL

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Review by EricJoy2048 — APPROVED** [review]
+1

**Pull request merged**